app/main.py

```diff
@@ -1,3 +1,36 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os
+from decimal import Decimal
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open(f"app{os.sep}config.json", "r") as file:
+        trip_data = json.load(file)
+
+    customers, shops = [], []
+    fuel_price = Decimal(str(trip_data["FUEL_PRICE"]))
+
+    for cust_characteristics in trip_data["customers"]:
+        cust_characteristics["car_characteristics"] = (
```

Comment:
cust_charactersitics is a very weird name for a variable. customer_data or similar would be better